onelogin_aws_cli/tests/test_configurationFile.py

```diff
@@ -0,0 +1,80 @@
+from io import StringIO
+from unittest import TestCase
+from unittest.mock import patch
+
+from onelogin_aws_cli.configuration import ConfigurationFile
+
+
+class TestConfigurationFile(TestCase):
+
+    def test_can_save_password_username_true(self):
+        str = StringIO()
+        str.write("""[defaults]
+save_password = false
+
+[profile_test]
+save_password = true""")
+        str.seek(0)
+        cfg = ConfigurationFile(str)
+        self.assertTrue(cfg.section("profile_test").can_save_password)
+
+    def test_can_save_password_username_false(self):
+        str = StringIO()
+        str.write("""[defaults]
+save_password = false
+
+[profile_test]
+save_password = false""")
+        str.seek(0)
+        cfg = ConfigurationFile(str)
+        self.assertFalse(cfg.section("profile_test").can_save_password)
+
+    def test_can_save_password_username_xor_1(self):
+        str = StringIO()
+        str.write("""[defaults]
+save_password = false
+
+[profile_test]
+save_password = true""")
+        str.seek(0)
+        cfg = ConfigurationFile(str)
+        self.assertTrue(cfg.section("profile_test").can_save_password)
+
+    def test_can_save_password_username_xor_2(self):
+        str = StringIO()
+        str.write("""[defaults]
+save_password = false
+
+[profile_test]
+save_password = false""")
+        str.seek(0)
+        cfg = ConfigurationFile(str)
+        self.assertFalse(cfg.section("profile_test").can_save_password)
+
+    def test_can_save_password_username_xor_3(self):
```

Comment:
For my OCD, could you add a test for defaults = True and empty elsewhere?

Reply:
Done,